google/cloud/aiplatform/v1beta1/schema/predict/instance_v1beta1/services/__init__.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
```

Comment:
We should exclude `schema/**/services/` in synth.py.

Reply:
Done.